protobuf/api/api.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+import "protobuf/definition/execution.proto";
+
+package api;
+option go_package = "github.com/mesg-foundation/core/protobuf/api";
+
+// Execution is the service for handle operations on Execution
+// such as get or stream.
+service Execution {
+  // Get returns a single Execution specified in a request.
+  rpc Get(GetRequest) returns (definition.Execution) {}
+
+  // Stream returns a stream of exeuction that satisfy createria
+  // specified in a request.
+  rpc Stream(StreamRequest) returns (stream definition.Execution) {}
+}
+
+// GetRequest defines request to retrive a single execution.
+message GetRequest {
```

Comment:
Need to add prefix or suffix `Execution` because its in package API

Reply:
Why do you want prefix or suffix here?

The standard practice is to use method name and Request suffix. If you want to add Execution to the name, then you have too add it to method too, so it will become `rpc GetExecution`.

Reply:
The message are not namespaced by the service in protobuf..

The rpc method will be `api.Execution.Get` but the message will be `api.GetRequest`.
It will create conflict with other rpc method on other service. eg `api.Service.Get` with message `api.GetRequest`

Reply:
Conversation moved to https://github.com/mesg-foundation/engine/issues/1013